Northstar.Custom/mod/scripts/vscripts/gamemodes/_gamemode_fw.nut

```diff
@@ -124,6 +124,10 @@ void function GamemodeFW_Init()
 	// _battery_port.gnut needs this
 	RegisterSignal( "BatteryActivate" )
 
+	// mp_thaw's intro spawnpoint have some problem, player can be stucked on the sky, so override it
+	if ( GetMapName() == "mp_thaw" )
+		SetSpawnpointGamemodeOverride( TEAM_DEATHMATCH )
```

Comment:
Does FW have custom spawn logic usually? I'm not sure how i feel about getting rid of that spawn logic for a given map, if its just the intro sequence could we set it back after?